### ISSUE TITLE
Dataset of ACDC

### ISSUE THREAD

**Issue by whduddhks** [Amshaker/unetr_plus_plus#19]
I downloaded the ACDC data set provided by README.md.
The dataset.json in the ACDC dataset contains information on training and testing, and the data that needs to be tested is also written in training, so I wonder if the training code excludes it.
Also, in nnFormer's dataset.json for ACDC dataset, the test data is not written in training, so I wonder why it was added.

In addition, if the test data was used for training, I wonder if the pre-trained model provided was learned in the same way.

Thank you.

**Comment by Amshaker** [comment]
Yes, it is ok. 

However, you can divide the training set into train and another validation set to tune the hyper-parameters, and then use the testing set for evaluation. In this way, you have to update the trainer file to divide them accordingly. 

Best regards,
Abdelrahman.
